### DISCUSSION TITLE
REST API

### DISCUSSION THREAD

**Discussion by darky** [JasperFx/marten#2799] [Ideas]
Would be great to have REST API for exposing **Marten** outside of .NET community

**Comment by mysticmind** [comment]
@darky This is a great idea, we already have this in mind to allow usage of Marten via other languages.

**Comment by mysticmind** [comment]
Moving this to discussions and will pull it back to issues when to get to work on this in the near future.

**Comment by oskardudycz** [comment]
I reopened this issue in https://github.com/JasperFx/marten/issues/2822, as I might be working on it in the next months.
